composites/SnippetPreview/components/Button.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+
+/**
+ * @summary:
+ * Returns a basic styled button
```

Comment:
Please don't split summary in two lines, remove the colon, and add a period at the end